src/Imaging/ImageGenerator.php

```diff
@@ -278,18 +278,53 @@ private function generate($image)
      */
     private function applyDefaultManipulations()
     {
-        $defaults = [];
+        $defaults = Config::get('statamic.assets.image_manipulation.defaults') ?? [];
+
+        if (is_array($defaults) && count($defaults) > 0) {
+            $defaults = $this->mapParamsToGlideApi($defaults);
+        }
 
         // Enable automatic cropping
         if (Config::get('statamic.assets.auto_crop') && $this->asset) {
             $defaults['fit'] = 'crop-'.$this->asset->get('focus', '50-50');
         }
 
-        // TODO: Allow user defined defaults and merge them in here.
-
         $this->server->setDefaults($defaults);
     }
 
+    /**
+     * Remap the params to the Glide API.
+     *
+     * @return array
+     */
+    private function mapParamsToGlideApi($params)
+    {
+        $legend = [
+            'background' => 'bg',
+            'brightness' => 'bri',
+            'contrast' => 'con',
+            'filter' => 'filt',
+            'format' => 'fm',
+            'gamma' => 'gam',
+            'height' => 'h',
+            'orientation' => 'or',
+            'pixelate' => 'pixel',
+            'quality' => 'q',
+            'sharpen' => 'sharp',
+            'width' => 'w',
+            'watermark' => 'mark',
```

Comment:
This is neat, but you can't use our tag aliases within presets like you mention in the comments. Should we just make people use the standard glide api params like the presets to be consistent?

Reply:
Actually we could probably add support for the aliases in the presets by using the same mapping you've just done.

Reply:
What about running the presets through the same method?

Reply:
Make it `public static` if other classes need it, since it's not a stateful helper 🔥

Reply:
Yeah I'd add a method to `GlideManager.php` - something like `normalizeParameters()` and move the mapping in there, so you could do `Statamic\Facades\Glide::normalizeParameters($defaults)` and `Glide::normalizeParameters($presets)` [on this line](https://github.com/statamic/cms/blob/15d1b80b18860b0b92f5756f1e917838ac1a8876/src/Imaging/Manager.php#L57).